CHANGELOG.md

```diff
@@ -113,6 +113,8 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - `supervisor-mode` parameter has been added for `generate-template` subcommand.
   (#1598)
 
+- Added maintenance command `restart-migration` to restart migration script. (#1728)
```

Comment:
This isn't a breaking change, is it? It should be placed into "New features" section.